<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DoctorSignUpForm</class>
 <widget class="QWidget" name="DoctorSignUpForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>567</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sign Up</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <spacer name="verticalSpacer_top">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QFrame" name="signUpFrame">
     <property name="styleSheet">
      <string notr="true">
        QFrame {
            background: #ffffff;
            border-radius: 16px;
            padding: 24px;
        }
        QLabel {
            font-size: 14px;
            color: #374151;
        }
        QLineEdit {
            border: 1px solid #d1d5db;
            border-radius: 8px;
            padding: 8px;
            font-size: 14px;
            background: #f9fafb;
        }
        QLineEdit:focus {
            border: 1px solid #3b82f6;
            background: #ffffff;
        }
        QPushButton {
            border: none;
            border-radius: 8px;
            padding: 10px;
            font-size: 15px;
            font-weight: bold;
        }
        QLineEdit, QCheckBox {
            color: black;
        }
        #signUpButton {
            background-color: qlineargradient(x1:0, y1:0, x2:1, y2:0,
                                             stop:0 #3b82f6, stop:1 #2563eb);
            color: white;
        }
        #signUpButton:hover {
            background-color: #1d4ed8;
        }
        #backToLoginButton {
            background: #e5e7eb;
            color: #111827;
        }
        #backToLoginButton:hover {
            background: #d1d5db;
        }
      </string>
     </property>
     <layout class="QVBoxLayout" name="formLayout" stretch="0,0,0,0,0,0,0,0,0,0">
      <item>
       <widget class="QLabel" name="titleLabel">
        <property name="styleSheet">
         <string notr="true">font-size:22px; font-weight:bold; margin:10px 0;</string>
        </property>
        <property name="text">
         <string>医生注册</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="warningLabel">
        <property name="styleSheet">
         <string notr="true">
            color: red;
            font-size: 12px;
            margin: 0px;
            padding: 0px;
         </string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="usernameEdit">
        <property name="placeholderText">
         <string>用户名</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="emailEdit">
        <property name="placeholderText">
         <string>邮箱</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="passwordEdit">
        <property name="echoMode">
         <enum>QLineEdit::EchoMode::Password</enum>
        </property>
        <property name="placeholderText">
         <string>密码</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="confirmPasswordEdit">
        <property name="echoMode">
         <enum>QLineEdit::EchoMode::Password</enum>
        </property>
        <property name="placeholderText">
         <string>确认密码</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="showPasswordCheckBox">
        <property name="text">
         <string>显示密码</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="emptyLabel">
        <property name="styleSheet">
         <string notr="true">
            margin: 0px;
            margin-bottom: 20px;
            padding: 0px;
            color: #3b82f6;
            font-size: 12px;
           </string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignRight</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="signUpButton">
        <property name="text">
         <string>注册</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="backToLoginButton">
        <property name="text">
         <string>返回登陆</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_bottom">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
